src/status_im/constants.cljs

```diff
@@ -200,3 +200,7 @@
 (def ^:const community-member-role-all 1)
 (def ^:const community-member-role-manage-users 2)
 (def ^:const community-member-role-moderator 3)
+
+(def local-pairing-connection-string-identifier "If any string begins with cs we know its a connection string. This is useful when we read QR codes we know it is a connection string
```

Comment:
could you pls elaborate? this looks weird, is it like some kind of comment inside string?

Reply:
I think @siddarthkay needs to reformat this piece of code because the value of the var is in the same line of the docstring. Unfortunately Github syntax coloring is quite limited and it doesn't understand the docstrings in def.

```clojure
(def local-pairing-connection-string-identifier
  "If any string begins with cs we know its a connection string. This is useful
  when we read QR codes we know it is a connection string if it begins with this
  identifier. An example of a connection string is ->
  cs2:5vd6J6:Jfc:27xMmHKEYwzRGXcvTtuiLZFfXscMx4Mz8d9wEHUxDj4p7:EG7Z13QScfWBJNJ5cprszzDQ5fBVsYMirXo8MaQFJvpF:3"
  "cs")
```

Reply:
> could you pls elaborate? this looks weird, is it like some kind of comment inside string?

Yes this is a docstring, the purpose is to define it in one place and explain why it exists, I use this string in 2 places on the codebase and this place seems like the best place to document why it exists.

Reply:
it's kinda ok for functions, but for such a short string it's little bit weird, but ok

Reply:
formatted